checks/check_extra780

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
It looks like this file was recently deleted in https://github.com/toniblyx/prowler/commit/8f83da985a7ba129317fa2716583f1181bfc4686 - if the intention is for it to be removed, I'll remove it from my branch. I just wasn't sure because I see a reference to extra780 in both `groups/group7_extras` and `groups/group14_elasticsearch`

Reply:
It doesn't have to be removed, it was my fault.